### PR TITLE
fix spectate unrealize

### PR DESCRIPTION
**Pull request by youbitchoc**

### CODE REVIEW

**Comment by youbitchoc** [comment]
spectator spawning stuff with beastmaster seems to be broken but i'm not too worried about fixing that

**Comment by 6fears7** [comment]
`spectator spawning stuff with beastmaster seems to be broken 
`
where do people come up with this stuff

File: Story/OnlineUIComponents/SpectatorOverlay.cs [hunk 1]
```diff
@@ -218,9 +218,23 @@ public override void Update()
 
             }
 
-
+            { // manually realize room of spectated player since we detach it from roomrealizer
```
Comment (6fears7) [inline Story/OnlineUIComponents/SpectatorOverlay.cs:221]: Thanks for fixing this

**Comment by youbitchoc** [comment]
> `spectator spawning stuff with beastmaster seems to be broken ` where do people come up with this stuff

i was testing it LOL

**Comment by youbitchoc** [comment]
superceded by #365

**Pull request closed without merging**